### PR TITLE
Allow both absolute and relative python install paths

### PR DESCRIPTION
**Pull request by ellert**
https://sft.its.cern.ch/jira/browse/ROOT-8692

### CODE REVIEW

**Comment by peremato** [comment]
@phsft-bot build

File: bindings/pyroot/CMakeLists.txt [hunk 1]
```diff
@@ -36,8 +36,13 @@ endif()
...
 foreach(pyfile ${pyfiles})
   install(FILES ${pyfile} DESTINATION ${runtimedir})
-  install(CODE "execute_process(COMMAND python -m py_compile \$ENV{DESTDIR}${runtimedir}/${pyfile})")
-  install(CODE "execute_process(COMMAND python -O -m py_compile \$ENV{DESTDIR}${runtimedir}/${pyfile})")
+  if(IS_ABSOLUTE ${runtimedir})
+    set(absruntimedir ${runtimedir})
+  else()
+    set(absruntimedir ${CMAKE_INSTALL_PREFIX}/${runtimedir})
+  endif()
+  install(CODE "execute_process(COMMAND python -m py_compile \$ENV{DESTDIR}${absruntimedir}/${pyfile})")
```
Comment (bastianbeischer) [inline bindings/pyroot/CMakeLists.txt]: Not strictly related to the patch, but shouldn't this also be ${PYTHON_EXECUTABLE} as below?

Comment (peremato) [inline bindings/pyroot/CMakeLists.txt]: Absolutely we should use ${PYTHON_EXECUTABLE}, specially for python3 :-)

**Comment by peremato** [comment]
@phsft-bot build

**Comment by peremato** [comment]
Merged. Thanks.

**Pull request closed without merging**